project_proposal/project_proposal.md

```diff
@@ -43,13 +29,17 @@ front-end to receive data via fetch calls in JavaScript it needs to talk with
 something that speaks HTTP and preferably delivers data in JSON.
 
 ### Objectives
+The group wants to create a dashboard solution, that gets the necessary
+information from the brewery machine to display information about current - and
+previous batches.
+
 The group wants to host the REST API and Website on a Linux based server on a
 fully-qualified domain name using the container technology Docker which should
 be configured via Docker Compose. Furthermore the group aims to setup a
-continuous integration pipeline for development and continuous delivery via 
-Watchtower.
+continuous integration pipeline for development via GitHub Actions and
+continuous delivery via Watchtower.
 
-#### Initial Requirement Analysis
+### Initial Requirement Analysis
```

Comment:
I am not sure to write here?

Reply:
The green one = ###

Reply:
Not sure what you mean?

Reply:
Wait, why are we on his pull request @kepet19 ?

Reply:
We are reviewers and should suggest what to write here?

Reply:
Initial Requirement Analysis:
- There should be and API to talk to the beer machine
- Front end that is writing in JavaScript and display with HTML and CSS.
- Database to hold data about batches. 
- Calculated the optimal speed, the beer machine should run at. the speed should be right to get maximum of products.

Reply:
huh? I just corrected the formatting, it's not my section

Reply:
Sorry did not see that. I merged it already.